### PR TITLE
Support running single test files with npx mocha - Closes #2930

### LINKED ISSUE #2930: Support running single test files with npx mocha
### Expected behavior
We should be able to run mocha tests with `npx mocha <file>` command.
### Actual behavior

We are not able to do that at the moment.

### PR DESCRIPTION
**Pull request by yatki**
### What was the problem?
We were not able to run mocha tests with `npx mocha <file>` command.
### How did I fix it?
I created a symlink for mocha.opts file under `test/mocha.opts` so mocha can read the options by default.
### How to test it?
Run:

`npx mocha framework/test/mocha/unit/components/cache/cache.js`

### Review checklist

* The PR resolves #2930 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: test/mocha.opts [hunk 1]
```diff
@@ -0,0 +1 @@
+../framework/test/mocha/mocha.opts
```
Comment (nazarhussain) [inline test/mocha.opts]: Does it work with IDEs e.g. Webstorm? What to do when root test directory will have its own test files?

Comment (yatki) [reply test/mocha.opts]: Please checkout the branch and try it with your IDE. I don't have it installed. Also, I think IDE related configuration shouldn't be application configuration's concern.

When the root project have its own tests, they would conflict with the current framework tests anyway. I think we can come up with a solution then.

Comment (nazarhussain) [reply test/mocha.opts]: I tried, its working with Webstorm.

Comment (nazarhussain) [reply test/mocha.opts]: @yatki The point is not far to introduce root level tests for the application implementation. So start thinking of the solution to this problem.

**Pull request merged**